src/scripts.js

```diff
@@ -311,22 +318,25 @@ function createMissingIngredientsModal(recipeID) {
 function closePopUp(event) {
   if (event.target.id === 'dismissButton') {
     popupSuccess.style.display = 'none';
+    popupError.style.display = 'none';
   }
 }
 
-function addRecipesToPantry(recipeID) {
+function addToPantry() {
+  const recipeId = Number(addIngredientsBtn.dataset.recipeId);
   toggleIngredientsBtn();
-  addAllIngredients(recipeID, store.user);
+  addAllIngredients(recipeId, store.user);
```

Comment:
- [x] Renamed the function to be more clear on what it does. 
- [x] Now getting the recipe id by grabbing it from the `ingredients btn dataset id`. 
- [x] Needing to convert id to a `number` on `line 326`